### PR TITLE
Tasl comleted

### PR DESCRIPTION
**Pull request by metja**
- [DEMO LINK](https://metja.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://metja.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,57 @@
...
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+  </head>
+  <body>
+    <div class="Card" data-qa="card">
```
Comment (SanyaBratashchuk) [inline src/index.html:11]:
```suggestion
    <div class="card" data-qa="card">
```
Do not use uppercase in css classes

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,57 @@
...
+      <img
+        class="Card-Image"
```
Comment (SanyaBratashchuk) [inline src/index.html:13]:
```suggestion
        class="Card__Image"
```

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,57 @@
...
+        src="./images/imac.jpeg"
+        alt="imac"
+      >
+
+      <h3 class="Card-Title">
```
Comment (SanyaBratashchuk) [inline src/index.html:18]:
```suggestion
      <h3 class="Card__Title">
```

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,57 @@
...
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <span class="Card-ProductCode">
```
Comment (SanyaBratashchuk) [inline src/index.html:22]:
```suggestion
      <span class="card__product-code">
```

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,57 @@
...
+        Product code: 195434
+      </span>
```
Comment (SanyaBratashchuk) [inline src/index.html:24]:
```suggestion
      <p class="Card-ProductCode">
        Product code: 195434
      </p>
```

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,57 @@
...
+          <li class="Stars-Star"></li>
+          <li class="Stars-Star"></li>
+          <li class="Stars-Star"></li>
+          <li class="Stars-Star"></li>
+        </ul>
+
+        <span class="ReviewsQuantity">
+          Reviews: 5
+        </span>
+      </div>
```
Comment (SanyaBratashchuk) [inline src/index.html:38]:
```suggestion
      <div class="card__reviews">
        <ul class="Stars Stars--4">
          <li class="Stars-Star"></li>
          <li class="Stars-Star"></li>
          <li class="Stars-Star"></li>
          <li class="Stars-Star"></li>
          <li class="Stars-Star"></li>
        </ul>

        <span class="card__reviews-text">
          Reviews: 5
        </span>
      </div>
```

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,57 @@
...
+
+      <a
+        href="#buy"
+        class="CardButton"
+        data-qa="hover"
+      >
+        Buy
+      </a>
```
Comment (SanyaBratashchuk) [inline src/index.html:54]:
```suggestion
      <div class="card__price">
        Price:

        <span class="Card__price-amount">
          $2,199
        </span>
      </div>

      <a
        href="#buy"
        class="card__button"
        data-qa="hover"
      >
        Buy
      </a>
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
You implementation doesn't follow BEM rules

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
There is problems with deploy? At least add tests and results screenshots.

File: src/styles/blocks/ProductCard.css [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card_product-code {
```
Comment (AlisaVlasova) [inline src/styles/blocks/ProductCard.css:23]:
```suggestion
.card__product-code {
```

File: src/styles/blocks/Stars.css [hunk 9]
```diff
@@ -0,0 +1,42 @@
+.card__stars-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```
Comment (AlisaVlasova) [inline src/styles/blocks/Stars.css:5]: це частина блоку card варто перенести у відповідний файл

File: src/styles/style.css [hunk 10]
```diff
@@ -0,0 +1,31 @@
+@import url(./blocks/ProductCard.css);
+@import url(./blocks/Stars.css);
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
```
Comment (AlisaVlasova) [inline src/styles/style.css:9]: add fallback fonts

File: src/styles/blocks/Stars.css [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+.stars--2 li:nth-child(n+3),
+.stars--3 li:nth-child(n+4),
+.stars--4 li:nth-child(n+5) {
+  background: url("../../images/star.svg") center no-repeat;
+}
+
+.card__reviews-quantity {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
```
Comment (AlisaVlasova) [inline src/styles/blocks/Stars.css:32]: це дефолтні стилі, їхможна не задавати явно

File: src/styles/blocks/ProductCard.css [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+  display: block;
+  color: #616070;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/ProductCard.css:25]: move color into variable

File: src/styles/blocks/ProductCard.css [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+  background-color: #00acdc;
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+  text-decoration: none;
+  text-align: center;
+  color: #fff;
+  border: 1px solid  transparent;
+  border-radius: 5px;
+  cursor: pointer;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/ProductCard.css:59]: link has this by default

File: src/styles/blocks/Stars.css [hunk 14]
```diff
@@ -0,0 +1,21 @@
...
+  height: 16px;
+  width: 16px;
+  background: url("../../images/star-active.svg") center no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 li:nth-child(n+1),
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/Stars.css:15]: don't style tags

File: src/styles/style.css [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+
+body,
+p,
+h1,
+h2,
+h3,
+h4,
+ul,
+span,
+img {
```
Comment (ivanchuk-yurii) [inline src/styles/style.css:22]: this elements don't have default `margin/padding`

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]
all of your tests should be passed, provide a screenshot

**Pull request closed without merging**